### PR TITLE
chore: Monolog v3 support

### PR DESCRIPTION
**Pull request by pdobrovolny**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "google/apiclient",
+    "name": "pdobrovolny/apiclient",
```
Comment (mfn) [inline composer.json:2]: 🤔

Comment (pdobrovolny) [reply composer.json:2]: oh sorry, my bad ... That belong to my fork. Apology

Comment (pdobrovolny) [reply composer.json:2]: by the way, the code seems to be compatible with monolog version 3. Perhaps You could fix those dependecies.

**Comment by bshaffer** [comment]
closing in favor of https://github.com/googleapis/google-api-php-client/pull/2268

**Pull request closed without merging**